cmake/Modules/FindCatch_EP.cmake

```diff
@@ -75,6 +91,8 @@ endif()
 if (CATCH2_FOUND AND NOT TARGET Catch2::Catch2)
   add_library(Catch2::Catch2 INTERFACE IMPORTED)
   set_target_properties(Catch2::Catch2 PROPERTIES
-    INTERFACE_INCLUDE_DIRECTORIES "${CATCH_INCLUDE_DIR}"
+    INTERFACE_INCLUDE_DIRECTORIES 
+      "${CATCH_INCLUDE_DIR}"
+      "${CATCH_INCLUDE_DIR}/.."
```

Comment:
It seems weird to need both directories. It feels like something's wrong with the setup.

Reply:
Entire section creating the target removed with changes in this file to search for specific version which should always have the needed targets for invokers of FindCatch_EP.